### DISCUSSION TITLE
Workshop content for Docker GenAI Stack

### DISCUSSION THREAD

**Discussion by ajeetraina** [collabnix/docker-aiml-stories#8] [General]
@bricefotzo We conducted a workshop on Docker GenAI Stack few months back. https://github.com/collabnix/docker-genai-workshop?tab=readme-ov-file It would be great if we can create a table with 4 tables and redirect to these workshop pages
https://genai-workshops-apac.netlify.app

**Comment by ajeetraina** [comment]
![Screenshot 2024-07-27 at 4 37 01 PM](https://github.com/user-attachments/assets/664fcd8c-9e0c-4c3b-af6d-1785a2c1978a)

**Reply by bricefotzo** [reply]
You want the labs to be added in grid format as videos right ?

**Reply by collabnix** [reply]
Yes. and it's already added by you.
